### PR TITLE
Boos 320/feature/logs

### PR DESCRIPTION
**Pull request by koutsilisk**
in this PR we introduce:
- Logs setup
- Logs custom configuration

Fix:
- Update ErrorHandler methods as classmethods

### CODE REVIEW

**Comment by koutsilisk** [comment]
@Chichilele, not sure about the handlers log level, since we are handling the errors seems ok to be debug (lowest log level), and only the 500 error handler to be critical (highest) . Need your thoughts on that! Thanks

File: backend/loggingConfig.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        "class": "logging.StreamHandler",
+        "formatter": "precise",
+        "stream": "ext://sys.stdout"
+      }
+    },
+    "root": {
+      "formatter": "precise",
+      "handlers": ["console"]
+    }
+  }
```
Comment (Chichilele) [inline backend/loggingConfig.json:23]: empty line

**Review by Chichilele — APPROVED** [review]

**Comment by Chichilele** [comment]
> @Chichilele, not sure about the handlers log level, since we are handling the errors seems ok to be debug (lowest log level), and only the 500 error handler to be critical (highest) . Need your thoughts on that! Thanks

Mhhh good point. How about 500 critical, anything related to authorisation/authentication info and the rest debug?

**Comment by koutsilisk** [comment]
> > @Chichilele, not sure about the handlers log level, since we are handling the errors seems ok to be debug (lowest log level), and only the 500 error handler to be critical (highest) . Need your thoughts on that! Thanks
> 
> Mhhh good point. How about 500 critical, anything related to authorisation/authentication info and the rest debug?

What I was thinking about, sounds good. Will update

**Pull request merged**